<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Treinar</class>
 <widget class="QMainWindow" name="Treinar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Treinar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="n1Edit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="n2Edit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="REdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
